blockly-core/tests/blockly_test.js

```diff
@@ -197,12 +197,112 @@ function test_initializeBlockSpace() {
   goog.dom.removeNode(container);
 }
 
-function test_initializeBlockSpace() {
+function test_initializeFunctionEditor() {
   var container = document.createElement('div');
   document.body.appendChild(container);
   Blockly.assetUrl = function(){return ''};
   Blockly.Css.inject(container);
   Blockly.mainBlockSpaceEditor = new Blockly.BlockSpaceEditor(container);
   Blockly.mainBlockSpace = Blockly.mainBlockSpaceEditor.blockSpace;
+  Blockly.focusedBlockSpace = Blockly.mainBlockSpace;
+  Blockly.hasTrashcan = true;
+  var functionalDefinitionXML = '<xml><block type="procedures_defnoreturn" editable="false"><mutation></mutation><title name="NAME">test-function</title></block></xml>'
+  var xml = Blockly.Xml.textToDom(functionalDefinitionXML);
+  Blockly.Xml.domToBlockSpace(Blockly.mainBlockSpace, xml);
+  Blockly.useModalFunctionEditor = true;
+  Blockly.functionEditor = new Blockly.FunctionEditor();
+  Blockly.functionEditor.autoOpenFunction('test-function');
+  Blockly.functionEditor.hideIfOpen();
   goog.dom.removeNode(container);
```

Comment:
I don't see any asserts. Does this test just make sure we dont get any exceptions?